.travis.yml

```diff
@@ -71,7 +71,7 @@ matrix:
           sources: [ 'ubuntu-toolchain-r-test' ]
           packages: [ 'libstdc++-5-dev' ]
     - os: osx
-      osx_image: xcode7.3
+      osx_image: xcode8.2
```

Comment:
Reason I upgraded was that clang++ from mason does not work with xcode 7.2: https://github.com/mapbox/mason/issues/389

We should likely test with both clang++ from mason and apple clang++ on OS X. Will hit that next.